content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Change Notes
+description: Istio 1.8 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** support for injecting `istio-cni` into `k8s.v1.cni.cncf.io/networks` annotation with preexisting value that uses JSON notation.
+  ([Issue #25744](https://github.com/istio/istio/issues/25744))
+
+- **Added** support for `INSERT_FIRST`, `INSERT_BEFORE`, `INSERT_AFTER` insert operations for `HTTP_ROUTE` in `EnvoyFilter`  ([Issue #26692](https://github.com/istio/istio/issues/26692))
```

Comment:
@GregHanson looks like this issue is still open. Should it be closed before release?

Reply:
the the pending api PR merged - closed it!